plugins/linux-capture/xcompcap-main.cpp

```diff
@@ -57,14 +58,42 @@ obs_properties_t *XCompcapMain::properties()
 		props, "capture_window", obs_module_text("Window"),
 		OBS_COMBO_TYPE_LIST, OBS_COMBO_FORMAT_STRING);
 
+	struct WinStrings {
```

Comment:
Not sure this is a good idea, but as far as I could find there's no way to create an `std::vector` with an anonymous struct. I considered using `std::vector<std::array<std::string, 3>>` but I think that's a bit confusing when accessing the elements of that array, the struct makes it obvious which is which.